### ISSUE TITLE
"redirection forbidden" error when using application generator

### ISSUE THREAD

**Issue by lgs** [fortuity/rails3-mongoid-devise#6]
applying the template I get this error: 

```
redirection forbidden: http://github.com/fortuity/rails3-mongoid-devise/raw/master/template.rb -> https://github.com/fortuity/rails3-mongoid-devise/raw/master/template.rb (RuntimeError)
```

here is the full stack http://pastie.org/1305919

**Comment by fortuity** [comment]
I've investigated the issue and I've entered a GitHub support ticket. Recently GitHub rolled out SSL for access to all GitHub pages and attempts to access via HTTP are redirected to HTTPS URL equivalents. Subsequently, it is no longer possible to generate a new Rails application using a GitHub-hosted application generator template. I've confirmed that the issue is not confined to my own application generator template. The same issue arises when using another template, such as the example in Ryan Bates's [App Templates in Rails 2.3](http://asciicasts.com/episodes/148-app-templates-in-rails-2-3). It seems that this issue results from Rails implementation issues apparently because 'open-uri' does not accommodate redirects from HTTP to HTTPS. Changing the template URL to specify HTTPS instead of HTTP apparently won't work because the Rails builder accepts only URLs beginning with "http" as a remote source. I will update as I learn more.
